src/XMakeTasks/project.json

```diff
@@ -1,9 +1,13 @@
 ﻿{
   "dependencies": {
-    "System.Collections.Immutable": "1.2.0"
+    "System.Collections.Immutable": "1.2.0",
```

Comment:
It doesn't with Newtonsoft.Json but it's technically nonconformant so this should be fixed (see also #1180).